README.md

```diff
@@ -88,6 +88,23 @@ Data curation brings the following guarantees.
 - The `targets` attribute contains the top most interfaces in an inheritance chain, unless bubbling conditions differ. For instance, the list may contain `{ "target": "Element", "bubbles": true }` but not also `{ "target": "HTMLElement", "bubbles": true }` since `HTMLElement` inherits from `Element`.
 - For target interfaces that belong to a bubbling tree, the `targets` attribute only contains the deepest interface in the bubbling tree on which the event may fire and bubble. For instance, the list may contain `{ "target": "HTMLElement", "bubbles": true }`, but not also `{ "target": "Document" }` since event would de facto fire at `Document` through bubbling.
 
+## Known consumers
+
+The following projects are known to use webref data:
+* MDN content management system, [yari](https://github.com/mdn/yari/)
+* [Web Platform Tests](https://github.com/web-platform-tests/wpt)
+* [Open Web Docs MDN Browser Compat Data collector](https://github.com/openwebdocs/mdn-bcd-collector)
+* [bikeshed](https://speced.github.io/bikeshed/), a specification authoring tool
+* [ReSpec](https://respec.org/docs/), a specification authoring tool
+* ["DOM" Bindings for TypeScript](https://github.com/microsoft/TypeScript-DOM-lib-generator)
+* [browser API bindings for Dart](https://github.com/dart-lang/)
+* the [Pulsar text editor](https://github.com/pulsar-edit/pulsar)
+* [PostCSS preset env plugin](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env)
+* [strudy](https://github.com/w3c/strudy)
+* [nodysseus](https://gitlab.com/ulysses.codes/nodysseus)
+* [WebIDLPedia](https://dontcallmedom.github.io/webidlpedia/)
+* [Webdex](https://dontcallmedom.github.io/webdex/)
+
```

Comment:
I'm thinking we could invite people to report projects we're not aware of. Something like:

```suggestion

Using webref data in a project that is not yet in the list? Let us know!
```